healthScore/requirements.txt

```diff
@@ -28,7 +30,7 @@ sqlparse==0.4.4
 termcolor==1.1.0
 typing_extensions==4.9.0
 tzdata==2024.1
-urllib3==1.26.18
+urllib3
```

Comment:
Please restore the version before pushing so that no deployment issues occur later